.github/workflows/master-to-relrc.yml

```diff
@@ -0,0 +1,25 @@
+name: Create rel/rc pull request
+on:
+  push:
+    branches:
+      - master
+jobs:
+  relrc-promotion:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: rel/rc
```

Comment:
I'm confused why we check out `rel/rc`, then fetch and hard reset to `master` - why not just check out master directly?

Reply:
It came from this example here: https://github.com/peter-evans/create-pull-request/blob/main/docs/examples.md#keep-a-branch-up-to-date-with-another

Reply:
<img width="1065" alt="image" src="https://user-images.githubusercontent.com/103960489/170789266-26ba9d72-f803-42e2-8344-d916d1513bc1.png">
We could either checkout `rel/rc` or specify it as `base` in `Create Pull Request` (CPR) according to the CPR docs.

Reply:
I've played around with a few different configurations on a private repo and the current one in the PR is the only one that properly triggers the pull request for me.